src/modules/hub.bicep

```
@@ -58,6 +60,15 @@ module storageAccount 'Microsoft.Storage/storageAccounts/deploy.bicep' = {
   }
 }
 
+module dataFactory 'Microsoft.DataFactory/factories/deploy.bicep' = {
+  name: 'factory'
```

nit: `dataFactory` in case there is some other "factory" that would overlap
```suggestion
  name: 'dataFactory'
```